autoloads/dome_connections.gd

```diff
@@ -45,6 +45,7 @@ func _add_connection(dome_a: Area2D, dome_b: Area2D):
 		# "distance": 100   # could use to determine squad travel time between domes
 	}
 	connections.append(new_connection)
+	dome_a.connections.append(dome_b)
```

Comment:
main problem i see with this is you're duplicating information -- keeping all the connection information in this script and querying as need be makes sure it's the source of truth.  This only really becomes important if you still wanted connections to be dynamic, like destroying or building a new connector

I wrote `get_dome_connections` further down this file for what you need in the movement

Reply:
Very good point. I think I can use that just fine. My bad on missing that. Got lost in the sauce of getting movement working!

Reply:
no worries, glad you did! key aspect :D